src/components/shared/Tag/Tag.tsx

```diff
@@ -54,6 +54,7 @@ const Tag = ({
         />
       )}
       {text ? (
+        // eslint-disable-next-line react/jsx-no-useless-fragment
```

Comment:
Why not refactor this to remove the fragment, so that the rule bypass will no longer be required ?